src/member/views.py

```diff
@@ -1449,8 +1450,10 @@ def get_success_url(self):
         )
 
 
-class ClientStatusSchedulerDeleteView(generic.DeleteView):
+class ClientStatusSchedulerDeleteView(
+        PermissionRequiredMixin, generic.DeleteView):
```

Comment:
@lingxiaoyang This is an example of the ugly python code when we have 79 characters as the line limit. It's my opinion!